recipe/meta.yaml

```diff
@@ -26,19 +26,22 @@ requirements:
     - m2w64-pkg-config  # [win]
     - pkg-config  # [not win]
     - posix  # [win]
+    - gnuconfig  # [unix]
     - {{ compiler('c') }}        # [unix]
     - {{ compiler('m2w64_c') }}  # [win]
     - make   # [unix]
   host:
     - xorg-inputproto
-    - xorg-libx11 1.6.*
+    - xorg-libx11 1.6.*  # [not (osx and arm64)]
+    - xorg-libx11 1.7.*  # [osx and arm64]
```

Comment:
We have been using slightly different boilerplate to do the libx11 pinning, namely:

https://github.com/conda-forge/xorg-libxfixes-feedstock/blob/0b7e78d73fbab3f3d93716b54a618e8b3fb4d055/recipe/meta.yaml#L41-L43

in the `host` section and and

https://github.com/conda-forge/xorg-libxfixes-feedstock/blob/0b7e78d73fbab3f3d93716b54a618e8b3fb4d055/recipe/meta.yaml#L49

in the `run` section. This format should be more future-proof for changes to the pinning. Would you mind changing this section to use the same template, including the same explanatory comment?